phys/module_pbl_driver.F

```diff
@@ -823,6 +835,8 @@ SUBROUTINE pbl_driver(                                          &
   flag_qg = .FALSE. ; IF ( PRESENT( F_QG ) ) flag_qg = F_QG
   flag_qnc = .FALSE. ; IF ( PRESENT( F_QNC ) ) flag_qnc = F_QNC !Used in CAMUWPBL
   flag_qni = .FALSE. ; IF ( PRESENT( F_QNI ) ) flag_qni = F_QNI !Used in CAMUWPBL
+  flag_qnwfa = .FALSE. ; IF ( PRESENT( F_QNWFA ) ) flag_qnwfa = F_QNWFA
+  flag_qnifa = .FALSE. ; IF ( PRESENT( F_QNIFA ) ) flag_qnifa = F_QNIFA
```

Comment:
This seems to be a good way to grab the value from an optional variable, and then this flag can be safely used in the mynn_bl_driver call.

Reply:
Pardon my confusion Dave, but is this just a comment? If so, thank you. If not, thanks anyway.

Reply:
@joeolson42 
Joe,
Yes, just a comment on recognizing what you have to do to use these fields that are both optional and my not be set even if they are present.
You're welcome